modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -36,65 +36,3 @@ You will not be able to fix issues that arise during the migration.
 Ensure you have created a backup before you start the migration.
 If you are running {productname} Server on a virtual machine, we recommend that you create a snapshot before you start.
 ====
-
-////
-== The Migration Script
-.Procedure: Migrating to {productname} 4.1
-
-The script runs as [systemitem]``root`` and performs these actions:
-
-
-. Stop the spacewalk services:
-+
-----
-spacewalk-service stop
-----
-
-. Move away the old repository configuration:
-+
-----
-mv /etc/zypp/repos.d /etc/zypp/repos.d.old
-mkdir /etc/zypp/repos.d
-----
-
-. Enable and refresh new repositories:
-+
-----
-zypper ar -n "Main Repository" http://download.opensuse.org/distribution/leap/15.2/repo/oss repo-oss
-zypper ar -n "Main Update Repository" http://download.opensuse.org/update/leap/15.2/oss repo-update
-zypper ar -n "Non-OSS Repository" http://download.opensuse.org/distribution/leap/15.2/repo/non-oss repo-non-oss
-zypper ar -n "Update Repository (Non-Oss)" http://download.opensuse.org/update/leap/15.2/non-oss/ repo-update-non-oss
-zypper ar -n "Uyuni Server Stable" https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/ uyuni-server-stable
-
-zypper ref
-----
-
-. Perform the distribution upgrade:
-+
-----
-zypper -n dup --allow-vendor-change
-----
-
-. To migrate the database to PostgreSQL 12 run the migrate script:
```

Comment:
And this needs to stay as well, this is not part of the script :-)

Reply:
ok, thanks for spotting!